packages/circle-react-elements/src/styles/tailwind.preset.ts

```diff
@@ -1,6 +1,6 @@
 import tailwindAnimate from 'tailwindcss-animate';
 
-const preset = {
+export const tailwindPreset = {
```

Comment:
I suggest to use more package-name related name here as consumer could use multiple presets in their app, so `circleElementsPreset` would be more accurate